pkgs/tools/admin/awscli2/default.nix

```diff
@@ -0,0 +1,83 @@
+{ lib
+, python3
+, groff
+, less
+, fetchFromGitHub
+}:
+
+let
+  py = python3.override {
+    packageOverrides = self: super: {
+      botocore = super.botocore.overridePythonAttrs (oldAttrs: rec {
+        version = "2.0.0dev30";
+        src = fetchFromGitHub {
+          owner = "boto";
+          repo = "botocore";
+          rev = "7967b9c5fb027c9962e0876f0110425da88b88f2";
+          sha256 = "18yn5l1f4nr1pih392qkyidnj7z10bd2cv7yx4qrl7asxxraspr9";
+        };
+      });
+      prompt_toolkit = super.prompt_toolkit.overridePythonAttrs (oldAttrs: rec {
+        version = "2.0.10";
+        src = oldAttrs.src.override {
+          inherit version;
+          sha256 = "1nr990i4b04rnlw1ghd0xmgvvvhih698mb6lb6jylr76cs7zcnpi";
+        };
+      });
+    };
+  };
+
+in with py.pkgs; buildPythonApplication rec {
+  pname = "awscli2";
+  version = "2.0.26"; # N.B: if you change this, change botocore to a matching version too
+
+  src = fetchFromGitHub {
+      owner = "aws";
+      repo = "aws-cli";
+      rev = version;
+      sha256 = "sha256:1ysmr17gbcj6vs9ywzwgvd9caxwxgg9bnfvvkyks4fii34ji5qq8";
+    };
```

Comment:
Indentation is messed up here; give it a run through https://github.com/nix-community/nixpkgs-fmt to fix automatically.

Reply:
Thanks for the feedback. I applied the suggestions.